.github/workflows/ROMROM-IOS-TESTFLIGHT.yaml

```
@@ -280,8 +276,16 @@ jobs:
       # Upload to TestFlight with Fastlane
       - name: Upload to TestFlight with Fastlane
         run: |
-          IPA_PATH=$(echo ios/build/ipa/*.ipa)
-          echo "Uploading $IPA_PATH"
+          # 절대 경로로 IPA 파일 찾기
+          IPA_PATH=$(find $GITHUB_WORKSPACE/ios/build/ipa -name "*.ipa" | head -1)
+          echo "Found IPA at: $IPA_PATH"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)